server/src/test/scala/za/co/absa/atum/server/api/controller/CheckpointControllerSpec.scala

```diff
@@ -61,7 +61,7 @@ class CheckpointControllerSpec extends ZIOSpecDefault with TestData {
     ).provide(
       CheckpointControllerImpl.layer,
       checkpointServiceMockLayer
-    )
+    ) @@ TestAspect.ifPropNotSet("runIntegration")
```

Comment:
*Self education:**
How this works, please?

Reply:
- This line is used to control the execution of tests based on system properties.
- The @@ operator is used to apply a test aspect to a spec.
- TestAspect.ifPropNotSet("runIntegration") is a test aspect that will skip the execution of the test if the system property "runIntegration" is not set.

So, if the system property "runIntegration" is not set, the tests will be skipped.

Reply:
Tip: Try to use string Integration in the name of the Integration test class names.